app/src/main/java/bubi/com/logintest/school/School.java

```diff
@@ -13,6 +13,10 @@
     private List<Teacher> teacherList = new ArrayList<>();
     private Headteacher headteacher;
 
+    public void setCapacity(int capacity) {
+        this.capacity = capacity;
```

Comment:
Excellent usage of **this**

Reply:
Java is the smart, not me :-)))

Reply:
Remind me to explain you how **this** actually works well

Reply:
ok